packages/cloudflare/worker/src/index-full.ts

```diff
@@ -0,0 +1,314 @@
+/**
+ * Cloudflare Sandbox Worker - Real World Test Implementation
+ * 
+ * This worker demonstrates the @computesdk/cloudflare provider in action.
+ * It creates sandboxes, executes code, and provides a REST API for testing.
+ */
+
+import { cloudflare } from "@computesdk/cloudflare";
+
+// CRITICAL: Export the Sandbox class for Durable Objects
+export { Sandbox } from "@cloudflare/sandbox";
```

Comment:
This doesn't appear right. we shouldn't need to import an cloudflare package w/ our @computesdk package.